ElectronClient/gui/NoteText.jsx

```diff
@@ -1475,12 +1480,25 @@ class NoteTextComponent extends React.Component {
 
 			// Add the number of newlines to the row
 			// and add the length of the final line to the column (for strings with no newlines this is the string length)
-			const newRange = {
-				start: { row: r.start.row + str1Split.length - 1,
-					column: r.start.column + str1Split[str1Split.length - 1].length },
-				end: { row: r.end.row + str1Split.length - 1,
-					column: r.end.column + str1Split[str1Split.length - 1].length },
-			};
+
+			var newRange = {};
+
+			if (byLine==false) {
+				// Correcting slection offset when text is surrounded by white spaces
```

Comment:
slection => selection

Also add more information about what the code does here please.

Reply:
in this PR #2582 we added the functionality to convert text into bold/italics/code even though they were surrounded by spaces,  so in wrapSelection we added this condition that is if byline==false (which means treat the selected text as a block not inline) implement this rule

<img width="848" alt="byline" src="https://user-images.githubusercontent.com/54576074/76683509-9e606b00-662a-11ea-8402-147ab055baa2.png">



Now  the issue for wrapping selected text inside markdown rule was created when you implemented this rule, (_because it counts the white spaces which creates an offset_) 

So only cases with byLine== false need the code for correction in selection offset, Hope this clears it up!